packages/fxa-auth-server/lib/signer.ts

```diff
@@ -2,15 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+import { JWTool, PrivateJWK } from '@fxa/vendored/jwtool';
 
-const { JWTool } = require('@fxa/vendored/jwtool');
-
-module.exports = function (secretKeyFile, domain) {
-  const key = JWTool.JWK.fromFile(secretKeyFile, { iss: domain });
+export default function (secretKeyFile: string, domain: string) {
```

Comment:
this won't work with the `require` call in key_server.js.  You'd need `require('../lib/signer').default(config.secretKeyFile, config.domain);`

Reply:
The tests definitely agree with you, when I look at the generated code, it looks the same as the JS though. Is there an explanation for this behavior somewhere?

Reply:
They don't look the same to me.  One is `exports = aFunction` and then other is `exports.default = aFunction`.  The type of exports is different.